microsoftgraph/microsoft-graph-explorer-v4
# Fix: Inputting beta version results in getting beta endpoint suggestions

## Overview

Fixes #2828 

### Demo

![image](https://github.com/user-attachments/assets/5c7f640b-54df-417e-a14f-3bb190d00de9)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout branch
* Load GE
* Erase current default query up until `https://graph.microsoft.com/` 
* Write out/select `beta` endpoint from suggestions
* Observe suggestions list is different from the v1.0 suggestions
